habitat-lab/habitat/tasks/rearrange/actions/actions.py

```diff
@@ -487,6 +489,9 @@ def __init__(self, *args, config, sim: RearrangeSim, **kwargs):
         self.base_vel_ctrl.controlling_ang_vel = True
         self.base_vel_ctrl.ang_vel_is_local = True
         self._allow_dyn_slide = self._config.get("allow_dyn_slide", True)
+        self._enable_rotation_check_for_dyn_slide = self._config.get(
```

Comment:
This `.get()` call is not needed as `enable_rotation_check_for_dyn_slide` defaults to True in `habitat-lab/habitat/config/default_structured_configs.py`.

Reply:
Agree, but I followed the convention here in actions.py. What do you think? Thank you!